Bottom: 6b02ad93a1218c6b2333862e6f32c2a5a9ca205c
Top:    f6a95b82093df74a0499d0ce3ccc35f41b14ebb3
Author: scott snyder <sss@karma>
Date:   2020-02-12 09:02:45 -0500

Refresh of gitignore

---

diff --git a/.gitignore b/.gitignore
new file mode 100644
index 0000000..b25c15b
--- /dev/null
+++ b/.gitignore
@@ -0,0 +1 @@
+*~
